ansible_collections/arista/avd/molecule/eos_cli_config_gen/intended/configs/host1.cfg

```diff
@@ -1103,6 +1162,14 @@ interface Port-Channel17
    ip address 192.0.2.3/31
    service-policy type pbr input MyPolicy
 !
+interface Port-Channel18
+   no switchport
+   ip ospf cost 99
+   ip ospf network point-to-point
+   ip ospf authentication message-digest
+   ip ospf area 0.0.0.12
+   ip ospf message-digest-key 55 md5 7 ABCDEFGHIJKLMNOPQRSTUVWXYZ
```

Comment:
please use the valid type-7 password. You can configure the password on device an it should convert it into type-7 then use the same here.

Reply:
Remove as all keys are covered already.